modules/appeals_api/config/routes.rb

```diff
@@ -198,64 +198,25 @@
     end
   end
 
-  appealable_issues_controller_path = '/appeals_api/appealable_issues/v0/appealable_issues'
-
```

Comment:
We had previously been duplicating the routes for contestable/appealable issues so that the url segments `contestable-issues` and `appealable-issues` would both work (while the renaming process continued). Renaming is complete, so I've removed the `contestable-issues` routes here.